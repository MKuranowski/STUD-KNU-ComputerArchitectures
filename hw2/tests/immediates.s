auipc, x31, 5800
addi x1, x0, 10
addi x1, x1, 20
addi x1, x1, -1
addi x2, x0, 2
slli x2, x2, 6
addi x3, x0, 45
srli x3, x3, 2
addi x4, x0, 819
xori x4, x4, 341
addi x5, x0, 819
andi x5, x5, 341
addi x6, x0, 819
ori x6, x6, 341
