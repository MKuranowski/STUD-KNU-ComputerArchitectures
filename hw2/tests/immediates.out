Processor's clock cycles: 14
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 56
x00 = 0
x01 = 29
x02 = 128
x03 = 11
x04 = 614
x05 = 273
x06 = 887
x07 = 0
x08 = 0
x09 = 0
x10 = 0
x11 = 0
x12 = 0
x13 = 0
x14 = 0
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = 0
x21 = 0
x22 = 0
x23 = 0
x24 = 0
x25 = 0
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = 23756800
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
